kermit/src/commonTest/kotlin/co/touchlab/kermit/KermitTest.kt

```diff
@@ -110,4 +220,81 @@ class KermitTest {
         logger.d { "Log Without Tag (Original Kermit)" }  // Ensuring first Kermit isn't affected by withTag
         testLogWriter.assertLast { tag == "Kermit" }
     }
+
+    @Test
+    fun testMutableLoggerConfig() {
+        val testConfig = object : MutableLoggerConfig {
+            override var minSeverity: Severity = Severity.Verbose
+            override var logWriterList: List<LogWriter> = listOf()
+        }
+        val logger = Logger(testConfig)
+        logger.d { "Message" }
+
+        val testLogWriter = getTestLogWriter()
+        testConfig.logWriterList = listOf(testLogWriter)
+        logger.d { "Message2" }
+
+        assertEquals(1, testLogWriter.logs.size)
+        testLogWriter.assertLast { message == "Message2" }
+        testLogWriter.assertLast { tag == "Kermit" }
+
+        testConfig.minSeverity = Severity.Info
+
+        val newLogger = logger.withTag("New Tag")
+        newLogger.d { "Message3" }
+        assertEquals(1, testLogWriter.logs.size)
+        testLogWriter.assertLast { message == "Message2" }
+        testLogWriter.assertLast { tag == "Kermit" }
+
+        newLogger.a { "Message4" }
+        assertEquals(2, testLogWriter.logs.size)
+        testLogWriter.assertLast { message == "Message4" }
+        testLogWriter.assertLast { tag == "New Tag" }
+    }
+
+    @Test
+    fun testMutableLoggerConfig_MultiThreading() {
+        val testLogWriter = getTestLogWriter()
+        val config = object : MutableLoggerConfig {
+            override var minSeverity: Severity = Severity.Verbose
+            override var logWriterList: List<LogWriter> = listOf(testLogWriter)
+        }
+        val logger = Logger(config)
+        val operations = 100
+        val ops = ThreadOperations {}
+        val threads = 10
+        repeat(operations) {
+            ops.exe {
+                logger.d { "message" }
+            }
+        }
+        ops.run(threads)
+        assertEquals(operations, testLogWriter.logs.size)
+    }
+
+//    @Test
```

Comment:
You could alternatively add `@Ignore` instead of commenting this out. Then you'd see mention of 1 ignored test in the output, which is good if you want the reminder and annoying if you don't.

Reply:
Will do before merging. WIll push one more commit